### DISCUSSION TITLE
Checkbox size

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2387] [Q&A] [answered]
Is there an easy way to make a `Checkbox` larger? By default, the font is smaller than the `FormItem` `Label` and `Input` controls.

**Comment by ElderJames** [comment, marked as answer]
@michaelcsikos You just need to modify the CSS to adjust the font size. But the font size of a `Label` and `Input` is 14px by default.
